### DISCUSSION TITLE
How to trigger username/password save popup for Keystore/Preferences/SecureStorage

### DISCUSSION THREAD

**Discussion by EthanBlank-Git** [dotnet/maui#9793] [Q&A]
Just a quick question regarding SecureStorage in Maui. On Android, I have been having varying results with a popup prompting to save the entered username/password when logging in within the app, however I have yet to experience anything similar on iOS. I was previously able to see a prompt on android in previous versions but as of the last couple updates it does not seem to prompt any more.

Is this just a bug in the current version of Maui? or is there an extra step that needs to be taken for saving a username/password to the KeyStore using SecureStorage? 

Thanks in advance!

**Comment by ShaneBridges** [comment]
I'm curious about this also. The popup seems to never show when a new user logs in when I was pretty sure it did before asking the user if they want to save the login information.

**Comment by jasells** [comment]
I  am also seeing this behavior.  It seems to be tied only to the username entry, so I'm guessing there's some effect/behavior/config not done correctly?  In my code base, it _has_ shown up occasionally, but not sure what changed between builds where it did show and didn't, because it has varried just between debug deploys with no changes.
